source/core/manifolds.cc

```diff
@@ -266,7 +266,13 @@ attach_manifolds_to_triangulation(
       if (manifolds.types[i] == Parameters::Manifolds::ManifoldType::spherical)
         {
           Point<spacedim> circleCenter;
-          circleCenter = Point<spacedim>(manifolds.arg1[i], manifolds.arg2[i]);
+          if (spacedim == 2)
```

Comment:
Is it worth converting the if to if constexpr, as the spacedim is a template?

Reply:
Definitely worth it. Done.